doc/040_backup.rst

```diff
@@ -483,6 +483,16 @@ The tags can later be used to keep (or forget) snapshots with the ``forget``
 command. The command ``tag`` can be used to modify tags on an existing
 snapshot.
 
+Scheduling backups
+******************
+
+Restic does not have a built-in way of scheduling backups, as it's a tool
+that runs when executed rather than a daemon. There are plenty of different
+ways to schedule backup runs on various different platforms, e.g. systemd
+and cron on Linux/BSD and Task Scheduler in Windows, depending on one's
+needs and requirements. When scheduling restic to run recurringly, please
+make sure to detect already running instances before starting the backup.
+
```

Comment:
This is a good explanation, but I'd like to see some examples like my comment at https://github.com/restic/restic/issues/519#issuecomment-570808000 (that got 11 upvotes):

> * [Backupninja](https://0xacab.org/riseuplabs/backupninja) with built-in Restic support
> * [Automatic restic backups using systemd services and timers](https://github.com/erikw/restic-systemd-automatic-backup)
> * [Restic backup configs](https://github.com/peperunas/configs/tree/master/restic)

Reply:
I understand, but we don't want to fill the documentation with pointers to that, for various reasons. There's a Recipes section in the forum for examples and scripts. Don't you think that anyone searching for the above keywords along with restic should be able to find plenty of examples from various sources and for various use cases?

Reply:
Fair enough, but let's say that here.  Something like, "For some ideas, see the Recipes section in the forum for examples and scripts." (with a link).